internal/schema/schema.go

```diff
@@ -130,22 +147,94 @@ type cacheEntry struct {
 	err    error
 }
 
+type notFoundErr struct {
+	Scheme string
+	Path   string
```

Comment:
```suggestion
	url      string // URL that failed to load
	fullPath string // Resolved file system path. For http/https it would be the full URL again.
```